.github/workflows/next-build.yml

```diff
@@ -7,7 +7,7 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-name: build-next
+name: Build and push Next Che E2E image to quai.io
```

Comment:
`build-next` is unified name across Eclipse Che component repos, e.g. https://github.com/eclipse-che/che-server/blob/main/.github/workflows/next-build.yml

So, I would keep it untouched.

Reply:
it is just a name which will be shown in Actions tab and has no effect on build actions:

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#name

I believe it should be clear what has built, why and where was pushed from this name. You don`t need to go through all workflow code to understand that.

![image](https://github.com/eclipse/che/assets/72726250/7eff7c6a-7a54-4ace-9cf8-88e61a214948)

Reply:
I see.

My point was that the this eclipse/che repo is not dedicated to e2e tests (looking at the root folder files), so I was not sure if it's eligible to modify `build-next` action.

Moreover - this action has outdated code which we don't need anymore:
```
    - name: Cache local Maven repository
      uses: actions/cache@v3
      with:
       path: ~/.m2/repository
       key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
       restore-keys: |
        ${{ runner.os }}-maven-      
    - name: Set up JDK 11
      uses: actions/setup-java@v3
      with:
        distribution: 'adopt'
        java-version: '11'
```

Anyway, we can consider movement to dedicated repo https://github.com/eclipse-che/che-tests

BTW: there is a TYPO in name: `quai.io` > `quay.io`.

Reply:
Fixed